### PR TITLE
healHospital.sqf Fixes - Don't Merge Yet please.

### PR DESCRIPTION
**Pull request by tkcjesse**
Lelel

### CODE REVIEW

File: Altis_Life.Altis/core/actions/fn_healHospital.sqf [hunk 1]
```diff
@@ -2,16 +2,19 @@
...
 titleText[localize "STR_NOTF_HS_Healing","PLAIN"];
 
 uiSleep 8;
 
 if(player distance (_this select 0) > 5) exitWith {titleText[localize "STR_NOTF_HS_ToFar","PLAIN"]};
 titleText[localize "STR_NOTF_HS_Healed","PLAIN"];
 player setDamage 0;
-SUB(CASH,100);
\ No newline at end of file
+SUB(CASH,100);
```
Comment (sargken) [inline Altis_Life.Altis/core/actions/fn_healHospital.sqf]: Could set that as config you can change ? Just an idea.

**Comment by tkcjesse** [comment]
Yes, I can add a config option for it... as well as the progress bar
let me finish my KoTH match. I'll add it later.

**Comment by sargken** [comment]
Not a problem just an idea for you.

**Comment by tkcjesse** [comment]
Why is kavala the only hospital with the heal option btw?

**Comment by Jawshy** [comment]
@tkcjesse, yeah, that's another thing; add NPCs to heal at the other two hospitals.

**Comment by tkcjesse** [comment]
I mean... Can i just add it to the medic shop npc or add a whole new guy?

**Comment by Jawshy** [comment]
I would say a new NPC. Kavala has it's own doctor; why not Athira and Pyrgos as well?

**Comment by tkcjesse** [comment]
To answer that question... because it'd require me to open editor :p lmao. Yeah... Imma redo my pull so its one commit...

**Pull request closed without merging**